### PR TITLE
Add image for hawkers

### PR DESCRIPTION
**Pull request by zatkiller**
Forgot to checkout to main branch first. Can just look at the last 2 commits

### CODE REVIEW

**Comment by ivantjh** [comment]
Can create a new branch from `main` and do the changes?

File: .gitignore [hunk 1]
```diff
@@ -30,7 +30,7 @@ coverage
 
 # Grunt intermediate storage (https://gruntjs.com/creating-plugins#storing-task-files)
 .grunt
-
+git
```
Comment (ivantjh) [inline .gitignore:33]: Why need gitignore `git` ah? The original `.git` should be automatically ignored.

**Review by ivantjh — CHANGES_REQUESTED** [review]
Need do link up the controllers too so that images can be uploaded to a single `HawkerCentre`

File: .gitignore (hunk 1 above)
Comment (zatkiller) [reply .gitignore:33]: I think i typed this by accident as I was trying to commit n push

**Comment by zatkiller** [comment]
New branch "new-add-hawker-image" created

**Pull request closed without merging**